util/devel/chplspell-dictionary

```diff
@@ -8313,7 +8314,6 @@ openfp
 opengroup
 openhigh
 openlow
-openmem
```

Comment:
Looks like there was a removal here as well that you'll likely want to undo.  Once that's done, I don't need to take a look again, thanks!